resources/assets/js/components/Assistant.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div :class="['notification', 'is-radiusless', 'animated', type ? `is-${type}` : '']" v-if="show">
+        <button class="delete touchable" @click="close()"></button>
+        <div class="title is-5" v-if="title">{{ title }}</div>
+        <nav class="columns">
+            <div class="column is-narrow">
+                <figure class="image is-64x64 is-pulled-left">
+                    <img src="/img/book.png" />
+                </figure>
+            </div>
+            <div class="column">
+                <slot><!-- the image tag and caption will be inserted here by Vue--></slot>
+            </div>
+        </nav>
+    </div>
+    <div class="level-right" v-else>
+        <a class="button is-medium is-primary is-outlined" @click="open()"><span class="icon"><i class="fa fa-question-circle"></i></span></a>
+    </div>
+</template>
+
+<script>
+    export default{        
+        data() {
+            return {
+                html: [],
+                show: true
+            };
+        },
+        props: {
+            type: String,
+            title: String,
+        },
+
+        mounted() {
+            if (typeof (Storage) !== "undefined" && localStorage.getItem("assistantShow") !== null) {
+                if (localStorage.getItem("assistantShow") === "false") {
```

Comment:
Would be really nice to be able to track each notification individually. The purpose of these notifications is to guide a first time user through the system, if they close the first one of these they may not realize/see the other notifications through the system.

Maybe use a 'key' prop?

Reply:
Good idea. What do you think about this? Note:Its currently only working on builds and modpacks page. For the show case.